beaker-vmware.gemspec

```diff
@@ -20,25 +20,11 @@ Gem::Specification.new do |s|
   # Testing dependencies
   s.add_development_dependency 'rspec', '~> 3.0'
   s.add_development_dependency 'rspec-its'
-  if RUBY_VERSION < "2.3"
-    s.add_development_dependency 'fakefs', '~> 0.6', '< 0.14'
-  else
-    s.add_development_dependency 'fakefs', '~> 0.6'
-  end
-  s.add_development_dependency 'rake', '~> 10.1'
-  s.add_development_dependency 'simplecov'
-  s.add_development_dependency 'pry', '~> 0.10'
-  s.add_development_dependency 'beaker', '~> 3.0'
```

Comment:
I'd actually argue `beaker` is a runtime dependency.

Reply:
I think all of the beaker plugins don't have beaker as a runtime dependency in their gemspec. They pull it in via the Gemfile for tests. At least one plugin mentioned that this is so to avoid dependency cycles. I don't care how we continue, but I think we should keep it identical across all the different plugins. I'm fine with the plugins pulling in beaker, beaker pulling in the plugins  or keeping it all separate and leave it to the user.

Reply:
I do think that the plugins should depend on Beaker because it can signal compatibility. Let's say we do another major version bump in Beaker that breaks plugin loading, there's no other way to specify it. It does mean we need to drop all beaker-* dependencies in beaker itself so I won't insist on it here. Just indicating the general direction.

Reply:
sounds good to me. I will keep that in mind for the next beaker release.